### ISSUE TITLE
Illegal instruction (core dumped) error when importing malaya

### ISSUE THREAD

**Issue by MagusWyvern** [huseinzol05/malaya#82]
When running the code:

```
import malaya
from pprint import pprint

malaya.generator.available_transformer()
```
Python sends back the error:

```
Traceback (most recent call last):
  File "/home/maguswyvern/Documents/Python/news.py", line 4, in <module>
    malaya.generator.available_transformer()
AttributeError: module 'malaya' has no attribute 'generator'
```

Is there something I'm doing wrong?

**Comment by MagusWyvern** [comment]
Malaya version - 4.2.3

I set up a new python venv and everything seems to be working fine now. Probably has something to do with an incorrect installation of Tensorflow.

Although I'm now having problems with loading the quantized dataset, how much memory should I allocate?

**Comment by huseinzol05** [comment]
Paste the error here, I saw some Windows got problem to load quantized models, maybe binary provided by Tensorflow for Windows not able to execute quantized model.

**Comment by MagusWyvern** [comment]
I was using Windows in the code given above but I forgot to copy the error log. I re-read the docs and docs say Windows not suitable. I switched to Ubuntu 20.04.2. Let me explain tell you step by step what I did.

I updated packages using
```
sudo apt-get update && sudo apt-get upgrade
```
After that I made a Python venv

```
python3 -m venv aaaa
```

I activated the venv
```
source aaaa/bin/activate
```

No errors so far, there's an (aaaa) at the start of each line in the terminal so I think it means I successfully entered my venv

I install malaya
```
pip3 install malaya
```
System response:
```
Collecting malaya
  Downloading malaya-4.2.3-py3-none-any.whl (2.2 MB)
     |████████████████████████████████| 2.2 MB 971 kB/s 
Collecting scikit-learn
  Downloading scikit_learn-0.24.2-cp38-cp38-manylinux2010_x86_64.whl (24.9 MB)
     |████████████████████████████████| 24.9 MB 494 kB/s 
Collecting tqdm
  Downloading tqdm-4.60.0-py2.py3-none-any.whl (75 kB)
     |████████████████████████████████| 75 kB 433 kB/s 
Collecting networkx
  Downloading networkx-2.5.1-py3-none-any.whl (1.6 MB)
     |████████████████████████████████| 1.6 MB 1.5 MB/s 
Collecting sentencepiece
  Downloading sentencepiece-0.1.95-cp38-cp38-manylinux2014_x86_64.whl (1.2 MB)
     |████████████████████████████████| 1.2 MB 3.2 MB/s 
Collecting herpetologist
  Downloading herpetologist-0.0.9-py3-none-any.whl (2.4 kB)
Collecting dateparser
  Downloading dateparser-1.0.0-py2.py3-none-any.whl (279 kB)
     |████████████████████████████████| 279 kB 3.0 MB/s 
Collecting ftfy
  Downloading ftfy-6.0.1.tar.gz (63 kB)
     |████████████████████████████████| 63 kB 144 kB/s 
Collecting scipy
  Downloading scipy-1.6.3-cp38-cp38-manylinux1_x86_64.whl (27.2 MB)
     |████████████████████████████████| 27.2 MB 349 kB/s 
Collecting unidecode
  Downloading Unidecode-1.2.0-py2.py3-none-any.whl (241 kB)
     |████████████████████████████████| 241 kB 706 kB/s 
Collecting numpy
  Downloading numpy-1.20.3-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (15.4 MB)
     |████████████████████████████████| 15.4 MB 15 kB/s 
Collecting tensorflow>=1.15
  Downloading tensorflow-2.5.0-cp38-cp38-manylinux2010_x86_64.whl (454.4 MB)
     |████████████████████████████████| 454.4 MB 11 kB/s 
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='pypi.org', port=443): Read timed out. (read timeout=15)")': /simple/requests/
Collecting requests
  Downloading requests-2.25.1-py2.py3-none-any.whl (61 kB)
     |████████████████████████████████| 61 kB 452 kB/s 
Collecting threadpoolctl>=2.0.0
  Downloading threadpoolctl-2.1.0-py3-none-any.whl (12 kB)
Collecting joblib>=0.11
  Downloading joblib-1.0.1-py3-none-any.whl (303 kB)
     |████████████████████████████████| 303 kB 877 kB/s 
Collecting decorator<5,>=4.3
  Downloading decorator-4.4.2-py2.py3-none-any.whl (9.2 kB)
Collecting memoization
  Downloading memoization-0.3.2-py3-none-any.whl (38 kB)
Collecting python-dateutil
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 506 kB/s 
Collecting tzlocal
  Downloading tzlocal-2.1-py2.py3-none-any.whl (16 kB)
Collecting regex!=2019.02.19
  Downloading regex-2021.4.4-cp38-cp38-manylinux2014_x86_64.whl (733 kB)
     |████████████████████████████████| 733 kB 2.3 MB/s 
Collecting pytz
  Downloading pytz-2021.1-py2.py3-none-any.whl (510 kB)
     |████████████████████████████████| 510 kB 1.2 MB/s 
Collecting wcwidth
  Downloading wcwidth-0.2.5-py2.py3-none-any.whl (30 kB)
Collecting opt-einsum~=3.3.0
  Downloading opt_einsum-3.3.0-py3-none-any.whl (65 kB)
     |████████████████████████████████| 65 kB 434 kB/s 
Collecting absl-py~=0.10
  Downloading absl_py-0.12.0-py3-none-any.whl (129 kB)
     |████████████████████████████████| 129 kB 1.9 MB/s 
Collecting gast==0.4.0
  Downloading gast-0.4.0-py3-none-any.whl (9.8 kB)
Collecting tensorflow-estimator<2.6.0,>=2.5.0rc0
  Downloading tensorflow_estimator-2.5.0-py2.py3-none-any.whl (462 kB)
     |████████████████████████████████| 462 kB 384 kB/s 
Collecting six~=1.15.0
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting keras-preprocessing~=1.1.2
  Downloading Keras_Preprocessing-1.1.2-py2.py3-none-any.whl (42 kB)
     |████████████████████████████████| 42 kB 109 kB/s 
Collecting wheel~=0.35
  Downloading wheel-0.36.2-py2.py3-none-any.whl (35 kB)
Collecting astunparse~=1.6.3
  Downloading astunparse-1.6.3-py2.py3-none-any.whl (12 kB)
Collecting protobuf>=3.9.2
  Downloading protobuf-3.17.0-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl (1.0 MB)
     |████████████████████████████████| 1.0 MB 2.8 MB/s 
Collecting flatbuffers~=1.12.0
  Downloading flatbuffers-1.12-py2.py3-none-any.whl (15 kB)
Collecting google-pasta~=0.2
  Downloading google_pasta-0.2.0-py3-none-any.whl (57 kB)
     |████████████████████████████████| 57 kB 537 kB/s 
Collecting grpcio~=1.34.0
  Downloading grpcio-1.34.1-cp38-cp38-manylinux2014_x86_64.whl (4.0 MB)
     |████████████████████████████████| 4.0 MB 1.6 MB/s 
Collecting tensorboard~=2.5
  Downloading tensorboard-2.5.0-py3-none-any.whl (6.0 MB)
     |████████████████████████████████| 6.0 MB 193 kB/s 
Collecting h5py~=3.1.0
  Downloading h5py-3.1.0-cp38-cp38-manylinux1_x86_64.whl (4.4 MB)
     |████████████████████████████████| 4.4 MB 1.1 MB/s 
Collecting termcolor~=1.1.0
  Downloading termcolor-1.1.0.tar.gz (3.9 kB)
Collecting typing-extensions~=3.7.4
  Downloading typing_extensions-3.7.4.3-py3-none-any.whl (22 kB)
Collecting wrapt~=1.12.1
  Downloading wrapt-1.12.1.tar.gz (27 kB)
Collecting keras-nightly~=2.5.0.dev
  Downloading keras_nightly-2.5.0.dev2021032900-py2.py3-none-any.whl (1.2 MB)
     |████████████████████████████████| 1.2 MB 726 kB/s 
Collecting idna<3,>=2.5
  Downloading idna-2.10-py2.py3-none-any.whl (58 kB)
     |████████████████████████████████| 58 kB 337 kB/s 
Collecting certifi>=2017.4.17
  Downloading certifi-2020.12.5-py2.py3-none-any.whl (147 kB)
     |████████████████████████████████| 147 kB 765 kB/s 
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.4-py2.py3-none-any.whl (153 kB)
     |████████████████████████████████| 153 kB 646 kB/s 
Collecting chardet<5,>=3.0.2
  Downloading chardet-4.0.0-py2.py3-none-any.whl (178 kB)
     |████████████████████████████████| 178 kB 2.2 MB/s 
Collecting google-auth<2,>=1.6.3
  Downloading google_auth-1.30.0-py2.py3-none-any.whl (146 kB)
     |████████████████████████████████| 146 kB 2.3 MB/s 
Collecting markdown>=2.6.8
  Downloading Markdown-3.3.4-py3-none-any.whl (97 kB)
     |████████████████████████████████| 97 kB 748 kB/s 
Collecting google-auth-oauthlib<0.5,>=0.4.1
  Downloading google_auth_oauthlib-0.4.4-py2.py3-none-any.whl (18 kB)
Collecting werkzeug>=0.11.15
  Downloading Werkzeug-2.0.0-py3-none-any.whl (288 kB)
     |████████████████████████████████| 288 kB 3.6 MB/s 
Collecting tensorboard-plugin-wit>=1.6.0
  Downloading tensorboard_plugin_wit-1.8.0-py3-none-any.whl (781 kB)
     |████████████████████████████████| 781 kB 1.1 MB/s 
Collecting tensorboard-data-server<0.7.0,>=0.6.0
  Downloading tensorboard_data_server-0.6.1-py3-none-manylinux2010_x86_64.whl (4.9 MB)
     |████████████████████████████████| 4.9 MB 841 kB/s 
Requirement already satisfied: setuptools>=41.0.0 in ./see/lib/python3.8/site-packages (from tensorboard~=2.5->tensorflow>=1.15->malaya) (44.0.0)
Collecting pyasn1-modules>=0.2.1
  Downloading pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
     |████████████████████████████████| 155 kB 1.2 MB/s 
Collecting rsa<5,>=3.1.4; python_version >= "3.6"
  Downloading rsa-4.7.2-py3-none-any.whl (34 kB)
Collecting cachetools<5.0,>=2.0.0
  Downloading cachetools-4.2.2-py3-none-any.whl (11 kB)
Collecting requests-oauthlib>=0.7.0
  Downloading requests_oauthlib-1.3.0-py2.py3-none-any.whl (23 kB)
Collecting pyasn1<0.5.0,>=0.4.6
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
     |████████████████████████████████| 77 kB 366 kB/s 
Collecting oauthlib>=3.0.0
  Downloading oauthlib-3.1.0-py2.py3-none-any.whl (147 kB)
     |████████████████████████████████| 147 kB 2.4 MB/s 
Building wheels for collected packages: ftfy, termcolor, wrapt
  Building wheel for ftfy (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/maguswyvern/Documents/see/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-otmmca2z/ftfy/setup.py'"'"'; __file__='"'"'/tmp/pip-install-otmmca2z/ftfy/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-mlms4bi7
       cwd: /tmp/pip-install-otmmca2z/ftfy/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for ftfy
  Running setup.py clean for ftfy
  Building wheel for termcolor (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/maguswyvern/Documents/see/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-otmmca2z/termcolor/setup.py'"'"'; __file__='"'"'/tmp/pip-install-otmmca2z/termcolor/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-ks63802n
       cwd: /tmp/pip-install-otmmca2z/termcolor/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for termcolor
  Running setup.py clean for termcolor
  Building wheel for wrapt (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/maguswyvern/Documents/see/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-otmmca2z/wrapt/setup.py'"'"'; __file__='"'"'/tmp/pip-install-otmmca2z/wrapt/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-hihn32ph
       cwd: /tmp/pip-install-otmmca2z/wrapt/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for wrapt
  Running setup.py clean for wrapt
Failed to build ftfy termcolor wrapt
ERROR: tensorflow 2.5.0 has requirement numpy~=1.19.2, but you'll have numpy 1.20.3 which is incompatible.
Installing collected packages: threadpoolctl, numpy, scipy, joblib, scikit-learn, tqdm, decorator, networkx, sentencepiece, memoization, herpetologist, six, python-dateutil, pytz, tzlocal, regex, dateparser, wcwidth, ftfy, unidecode, opt-einsum, absl-py, gast, tensorflow-estimator, keras-preprocessing, wheel, astunparse, protobuf, flatbuffers, google-pasta, grpcio, pyasn1, pyasn1-modules, rsa, cachetools, google-auth, markdown, oauthlib, idna, certifi, urllib3, chardet, requests, requests-oauthlib, google-auth-oauthlib, werkzeug, tensorboard-plugin-wit, tensorboard-data-server, tensorboard, h5py, termcolor, typing-extensions, wrapt, keras-nightly, tensorflow, malaya
    Running setup.py install for ftfy ... done
    Running setup.py install for termcolor ... done
    Running setup.py install for wrapt ... done
Successfully installed absl-py-0.12.0 astunparse-1.6.3 cachetools-4.2.2 certifi-2020.12.5 chardet-4.0.0 dateparser-1.0.0 decorator-4.4.2 flatbuffers-1.12 ftfy-6.0.1 gast-0.4.0 google-auth-1.30.0 google-auth-oauthlib-0.4.4 google-pasta-0.2.0 grpcio-1.34.1 h5py-3.1.0 herpetologist-0.0.9 idna-2.10 joblib-1.0.1 keras-nightly-2.5.0.dev2021032900 keras-preprocessing-1.1.2 malaya-4.2.3 markdown-3.3.4 memoization-0.3.2 networkx-2.5.1 numpy-1.20.3 oauthlib-3.1.0 opt-einsum-3.3.0 protobuf-3.17.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 python-dateutil-2.8.1 pytz-2021.1 regex-2021.4.4 requests-2.25.1 requests-oauthlib-1.3.0 rsa-4.7.2 scikit-learn-0.24.2 scipy-1.6.3 sentencepiece-0.1.95 six-1.15.0 tensorboard-2.5.0 tensorboard-data-server-0.6.1 tensorboard-plugin-wit-1.8.0 tensorflow-2.5.0 tensorflow-estimator-2.5.0 termcolor-1.1.0 threadpoolctl-2.1.0 tqdm-4.60.0 typing-extensions-3.7.4.3 tzlocal-2.1 unidecode-1.2.0 urllib3-1.26.4 wcwidth-0.2.5 werkzeug-2.0.0 wheel-0.36.2 wrapt-1.12.1
```

Then I started python3 and wrote this single line
```
import malaya
```
The interpreter responded with 
```
Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import malaya
Illegal instruction (core dumped)
```

That's it, what should I do? I'm confused

**Comment by MagusWyvern** [comment]
I noticed the wrong version of numpy is installed so I uninstalled numpy 1.20.3 and installed numpy 1.19.5.

Same results

Ubuntu also reported an error
![image](https://user-images.githubusercontent.com/72914828/118231926-5fde6e80-b4c2-11eb-8514-f351cdf7ea2a.png)

**Comment by huseinzol05** [comment]
It can be OS issue, can try Ubuntu <= 20.04, not included 20.04.2. I cannot replicate the same issue in Ubuntu 16.04, 18.04, 20.04.
